Formula/epa-ng.rb

```diff
@@ -0,0 +1,24 @@
+class EpaNg < Formula
+  desc "Massively parallel phylogenetic placement of genetic sequences"
+  homepage "https://github.com/Pbdas/epa-ng"
+  url "https://github.com/Pbdas/epa-ng/archive/v0.3.5.tar.gz"
+  sha256 "0aa2aa9ffad0d713c42ea43330798a4d701ece883ced617da07218b7c697df41"
+
+  depends_on "cmake" => :build
+  unless OS.mac?
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+  end
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
```

Comment:
```suggestion
      system "make", "install"
```
Can `make install` be used rather than `bin.install Dir["bin/epa-ng*"]`?

Reply:
Their Makefile doesn't have an install command. This is much like the `raxml-ng` recipe. Because there is only the one single binary, I've edited the command to reflect this.